### PR TITLE
CLI prototype almost done

### PR DESCRIPTION
**Pull request by emilejbm**
build, install, test, URL_FILE commands are recognized. URL_FILE takes in txt file and prints content within file. Not sure what to install for install command. Missing the test suite to test on.

### CODE REVIEW

**Comment by Ashassins** [comment]
Resolves Issue #9

File: commands/CLI/test.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package team23
+
+import (
+	"fmt"
+	//"github.com/emilejbm/461_1_23/functionality/CLI"
+	"github.com/spf13/cobra"
+)
+
+var testCmd = &cobra.Command{
```
Comment (Ashassins) [inline commands/CLI/test.go:9]: I would change the filename. Maybe start organizing the test suite folder and structure?

File: commands/CLI/url_file.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package team23
+
+import (
+	"os"
+	"github.com/spf13/cobra"
+	"github.com/emilejbm/461_1_23/functionality/CLI"
+)
+
+var urlfileCmd = &cobra.Command{
```
Comment (Ashassins) [inline commands/CLI/url_file.go:9]: please add some comments on how you expect url_file links to API, etc.

File: functionality/CLI/team23.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+func test(input string) (int){
+	return 0
+}
+
+func READ_url_file(input []string) (int) {
+	filename := os.Args[2]
+
+	contents, err := os.ReadFile(filename)
+    if err != nil {
```
Comment (Ashassins) [inline functionality/CLI/team23.go:17]: Can we throw an error instead?

**Review by Ashassins — CHANGES_REQUESTED** [review]

**Comment by benschwartz9** [comment]
Merged through https://github.com/19chonm/461_1_23/pull/19

**Pull request closed without merging**